File: apps/hilltop_custom/hilltop_custom/hilltop_custom/report/script_report/script_report.py
# Copyright (c) 2024, obaid and contributors
# For license information, please see license.txt

# import frappe


def execute(filters=None):
	columns=[{
		"fieldname":"name",
		"label": "Name",
		"fieldtype":"Data",
		"width":100
	}]

	data = [{"name":"Kiranmai"},{"name":"Chinuu"},{"name":"Jhanu"}]
	return columns, data

File: apps/hilltop_custom/hilltop_custom/hilltop_custom/report/script_report/script_report.js
// Copyright (c) 2024, obaid and contributors
// For license information, please see license.txt

frappe.query_reports["Script Report"] = {
	"filters": [
		{
			"fieldname": "enable",
			"fieldtype": "Check",
			"label": "Enable",
			"mandatory": 0,
			"wildcard_filter": 0
		   }
	]
};
